data/css/escola.scss

```diff
@@ -33,8 +33,8 @@ $display-font: 'Raleway Black' !default;
       &__logo {
         -EknDynamicLogo-sizing: "size-min";
         color: white;
-        min-height: 120px;
-        min-width: 400px;
+        min-height: 200px;
+        min-width: 600px;
```

Comment:
If the logo is min 600px then I'm pretty sure this will also break composite 720x480.